app/utils/PresentationNotificationAnswersHelper.scala

```diff
@@ -32,6 +32,7 @@ import java.time.LocalDate
 class PresentationNotificationAnswersHelper(
   userAnswers: UserAnswers,
   departureId: String,
+  borderModes: Seq[BorderMode],
```

Comment:
Do we want to make more Granular Helpers, if we are pulling a lot of ref date - then this could get bulky